README.md

```diff
@@ -108,15 +108,15 @@ chown pe-puppet /etc/puppetlabs/puppet/ssl/satellite/puppet.example.com-puppet-c
 chown pe-puppet /etc/puppetlabs/puppet/ssl/satellite/puppet.example.com-puppet-client.key
 ```
 
-f. In the Satellite UI, go to *Administer -> Settings -> Auth* and set the `restrict_registered_puppetmasters` parameter to `true`. Additionally, add your Puppet master's FQDN to the `trusted_puppetmaster_hosts` array on the same page; for example, `[puppet.example.com]`.
+f. In the Satellite UI, go to *Administer -> Settings -> Auth* and set the `restrict_registered_puppetmasters` parameter to `true`. Additionally, add your Puppet server's FQDN to the `trusted_puppetserver_hosts` array on the same page; for example, `[puppet.example.com]`.
```

Comment:
`trusted_puppetserver_hosts` doesn't exist. Also, `trusted_puppetmaster_hosts` has been renamed to `trusted_hosts` in 1.18. That should cover all supported Satellite versions.

Reply:
have changed it to refer to trusted_hosts